web3/_utils/module_testing/eth_module.py

```diff
@@ -2315,6 +2342,32 @@ def test_eth_max_priority_fee(self, w3: "Web3") -> None:
         max_priority_fee = w3.eth.max_priority_fee
         assert is_integer(max_priority_fee)
 
+    def test_eth_max_priority_fee_with_fee_history_calculation_error_dict(
+        self, w3: "Web3"
+    ) -> None:
+        fail_max_prio_middleware = construct_error_generator_middleware(
+            {
+                RPCEndpoint("eth_maxPriorityFeePerGas"): lambda *_: {
+                    "error": {
+                        "code": -32601,
+                        "message": "some message",
+                    }
+                }
+            }
+        )
+        w3.middleware_onion.add(
+            fail_max_prio_middleware, name="fail_max_prio_middleware"
+        )
+
+        with pytest.warns(
+            UserWarning,
+            match="There was an issue with the method eth_maxPriorityFeePerGas."
```

Comment:
I'm not sure if there is a comma missing here?

Reply:
In the actual grammar of the messaging or just between the two match lines?

Reply:
Ah now I understand it, multiline concatenation. Hadn't seen this, was expecting `\` or `+`

Reply:
> Ah now I understand it, multiline concatenation. Hadn't seen this, was expecting `\` or `+`

For sure... sometimes using parentheses help with readability too. Might be a nice nit.

```py
with pytest.warns(
    UserWarning,
    match=(
        "There was an issue with the method eth_maxPriorityFeePerGas. "
        "Calculating using eth_feeHistory."
    ),
)
```